app/controllers/api/v0/listings_controller.rb

```diff
@@ -51,14 +52,6 @@ def show
         set_surrogate_key_header @listing.record_key
       end
 
-      def create
-        super
-      end
-
-      def update
-        super
-      end
```

Comment:
these should be superflous as they don't actually do anything, Rubocop removes them now, tests passed as well